core/common/src/main/java/alluxio/master/PollingMasterInquireClient.java

```diff
@@ -130,7 +130,8 @@ private InetSocketAddress getAddress() {
         LOG.debug("Timeout while connecting to {}", address);
         continue;
       } catch (AlluxioStatusException e) {
-        throw new RuntimeException(e);
+        LOG.debug("Exception while connecting to {}. error: {}", address, e.getMessage());
```

Comment:
Is there a way to aggregate the error here and log/throw it if no valid master is selected?

Reply:
that already happens elsewhere: https://github.com/Alluxio/alluxio/pull/12376/files#diff-7f60cdbf4f4b4313bee9dd5a28c81ce3577fe6e120c8f8eab522386d7085afe0R112